assets/text/nes_message_data_static.c

```diff
@@ -5,8 +5,15 @@
 #define DEFINE_MESSAGE(textId, type, yPos, nesMessage, gerMessage, fraMessage) \
     const char _message_##textId##_nes[sizeof(nesMessage)] = { nesMessage END };
 
-#define DEFINE_MESSAGE_FFFC
-
 #include "assets/text/message_data.enc.h"
 
-#undef DEFINE_MESSAGE_FFFC
+DEFINE_MESSAGE(0xFFFC, 0, 0,
```

Comment:
I agree, but

the reason for this change is technical: this PR cpp's `assets/message_data.h` before .enc-oding it, because otherwise files included by `assets/message_data.h` are not cpp'd

Note having `assets/message_data.h` include a .enc.h version of the extracted assets is not viable, as I want to support modding text by placing text in `assets/message_data.h` or by including files from there

Reply:
I missed a step in the explanation. The fact cpp runs "early" means the `#ifdef` are resolved early as well, and the defines around .enc includes have no effect

Reply:
Should now be addressed by essentially reverting the changes, but using a `DEFINE_MESSAGE_NES` macro instead of ifdefs (you suggested `DEFINE_MESSAGE_FFFC` but I figured NES looked better and is correct)